src/DotNetty.Transport/Channels/Sockets/ISocketDatagramChannel.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Transport.Channels.Sockets
+{
+    using System.Net;
+    using System.Net.NetworkInformation;
+    using System.Threading.Tasks;
+    using DotNetty.Common.Concurrency;
+
+    public interface ISocketDatagramChannel : ISocketChannel
```

Comment:
My comment was about class only :)
`ISocketChannel` should be replaced with `IChannel`, ISocketDatagramChannel` -> `IDatagramChannel`.